docs/reference/dlm/apis/get-lifecycle.asciidoc

```diff
@@ -0,0 +1,143 @@
+[[dlm-get-lifecycle]]
+=== Get the lifecycle of a data stream
+++++
+<titleabbrev>Get Data Stream Lifecycle</titleabbrev>
+++++
+
+experimental::[]
+
+Gets the lifecycle of a set of data streams.
+
+[[dlm-get-lifecycle-request]]
+==== {api-request-title}
+
+`GET _data_stream/_lifecycle`
```

Comment:
@dakrone what do you think? (note that this also applies to the `PUT` and `DELETE` methods)

Reply:
I mainly wanted to be inline with existing conventions. I do not have a strong opinion about this honestly. I am slightly in favor of not supporting it as well because then the path is more predictable.